pkg/hs/urbit-king/lib/Urbit/Vere/Eyre/Wai.hs

```diff
@@ -1,71 +1,77 @@
-{-|
-  WAI Application for `eyre` driver.
-
-  # Request Lifecycles
-
-  - Requests come in, are given an identifier and are passed to a callback.
-
-  - When requests timeout, the identifier is passed to anothing callback.
-
-  - The server pulls response actions, and passes them to the associated
-    request.
--}
-
+{-# LANGUAGE LambdaCase #-}
```

Comment:
Looks like a linter mangled this file? Busted coms, indentation and other stylistic changes all over the place.

Reply:
ah i thought cleared those out. will fix.